tests/test_views.py

```diff
@@ -63,33 +64,46 @@ def test_get_start_end_date_for_CPI_NONFOOD_rog_returns_proper_dates(self):
 
 # TODO: these test should relate to something else not covered in query.py
 
-# class TestGetResponseDatapoints(TestCaseBase):
-#
-#    data_dicts = [{"date": "1999-01-31", "freq": "m", "name": "CPI_ALCOHOL_rog", "value": 109.7},
-#                  {"date": "1999-01-31", "freq": "m",
-#                      "name": "CPI_FOOD_rog", "value": 110.4},
-#                  {"date": "1999-01-31", "freq": "m", "name": "CPI_NONFOOD_rog", "value": 106.2}]
-#
-#    def _make_sample_datapoints_list(self):
-#        return [Datapoint(**params) for params in self.data_dicts]
-#
-#    def test_json_serialising_is_valid(self):
-#        data = self._make_sample_datapoints_list()
-#        response = get_datapoints_response(data, 'json')
-#        parsed_json = json.loads(response.data)
-#        self.assertEqual(self.data_dicts, parsed_json)
-#
-#    def test_csv_serialising_is_valid(self):
-#        data = self._make_sample_datapoints_list()
-#        response = get_datapoints_response(data, 'csv')
-#        csv_string = str(response.data, 'utf-8')
-#        self.assertEqual(
-#            ',CPI_ALCOHOL_rog\n1999-01-31,109.7\n1999-01-31,110.4\n1999-01-31,106.2\n', csv_string)
-#
-#    def test_invalid_output_format_should_fail(self):
-#        data = self._make_sample_datapoints_list()
-#        with self.assertRaises(CustomError400):
-#            get_datapoints_response(data, 'html')
+class TestDatapointsAPI(TestCaseBase):
+
+    data_dicts = [{
+        "date": "2016-06-01",
+        "freq": "d",
+        "name": "USDRUR_CB",
+        "value": 65.9962
+        },
+        {
+        "date": "2016-06-02",
+        "freq": "d",
+        "name": "USDRUR_CB",
+        "value": 66.6156
+        },
+        {
+        "date": "2016-06-03",
+        "freq": "d",
+        "name": "USDRUR_CB",
+        "value": 66.7491
+    }]
+
+    data_csv_string = ",USDRUR_CB\n2016-06-01,65.9962\n2016-06-02,66.6156\n2016-06-03,66.7491\n"
+
+    def _get_formatted_response(self, response_format):
```

Comment:
Get formatted is nit a great name, because youhave format args and there is no 'unformatted' response.